### ISSUE TITLE
Doesn't work for Anki 2.1.49 :( 

### ISSUE THREAD

**Issue by Artset08** [sachingooo/anki-congrats-confetti#1]
I'm med student too and I was really happy to find this kind of addon... But it doesn't work for my anki version :( (I already restarted anki several times)

**Comment by sachingooo** [comment]
Yeah, the canvas element on the overview/congrats page doesn't get added in the right place in 2.1.49 😕. I can look into making this compatible with 2.1.49+ with some version-specific handling, but I'd honestly recommend upgrading to 2.1.56 (newest version). Most addons are now supporting 2.1.54+ after the layout and web handling changed in that version.
